packages/api/test/integration.test.ts

```diff
@@ -54,11 +54,11 @@ describe('integration tests', function () {
         version: 'oa_citations',
       };
 
-      const res = await api(usptoSpec).post('/{dataset}/{version}/records', body, metadata);
-      expect(res.uri).to.equal('/anything/v1/oa_citations/records');
-      expect(res.requestBody).to.equal('criteria=propertyName%3Avalue');
-      expect(res.headers).to.have.deep.property('content-type', 'application/x-www-form-urlencoded');
-      expect(res.headers).to.have.a.customUserAgent;
+      const { data } = await api(usptoSpec).performSearch(body, metadata);
+      expect(data.uri).to.equal('/anything/v1/oa_citations/records');
+      expect(data.requestBody).to.equal('criteria=propertyName%3Avalue');
+      expect(data.headers).to.have.deep.property('content-type', 'application/x-www-form-urlencoded');
+      expect(data.headers).to.have.a.customUserAgent;
```

Comment:
I'm confused on how these tests pass 🧐 isn't `data` here the response body and shouldn't the headers be separate?
```suggestion
      const { data, headers } = await api(usptoSpec).performSearch(body, metadata);
      expect(data.uri).to.equal('/anything/v1/oa_citations/records'); // not sure what this should be
      expect(data).to.equal('criteria=propertyName%3Avalue');
      expect(headers).to.have.deep.property('content-type', 'application/x-www-form-urlencoded');
      expect(headers).to.have.a.customUserAgent;
```

Reply:
These tests are the way they are because I'm mocking the response to return data that was sent, this way I can assert that we're sending the right things.

Reply:
Ah I totally didn't think about the fact that mocking could be a thing here 🤦🏽